src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/List.js

```diff
@@ -146,13 +148,13 @@ class List extends React.Component<Props> {
     }
 
     componentDidUpdate(prevProps: Props) {
-        const {adapters, store} = this.props;
+        const {adapters, store, paginated} = this.props;
         if (!equal(adapters, prevProps.adapters)) {
             this.validateAdapters();
         }
 
         if (store !== prevProps.store) {
-            store.updateLoadingStrategy(new this.currentAdapter.LoadingStrategy());
+            store.updateLoadingStrategy(new this.currentAdapter.LoadingStrategy(paginated));
```

Comment:
as discussed, passing an object like `{paginated: paginated}` here would be easier to extend in the future 🙂

Reply:
not sure if there is a test for this, would be nice to check the passed value there